pwm/ad_notify_passwd_expire.sls

```diff
@@ -0,0 +1,46 @@
+{%- if pillar['ad_notify_passwd_expire'] == 1 %}
```

Comment:
Generally the convention for a formula is to namespace the pillar keys it uses under `<formula-name>:lookup`. In this case, would rewrite this as:

```suggestion
{%- if salt.pillar.get('pwm:lookup:ad_notify_passwd_expire') %}
```

And then any truthy value will match.